url/resources/urltestdata.json

```diff
@@ -8164,6 +8164,11 @@
     "base": "about:blank",
     "failure": true
   },
+  {
+    "input": "http://foo.09..",
```

Comment:
In my whatwg-url implementation this does not fail. I guess you are interpreting the spec different from how it is strictly written? Where it only ever considers the last item after splitting, instead of your interpretation of considering the last non-empty item. So this might be a spec bug.

/cc @MattMenke2 @annevk.

Reply:
Yeah, I think with the current spec it should not fail.  It's strange to me that one dot after the number fails but two does not.  In any case, I think I'm going to change this test for now.